src/PrestaShopBundle/Translation/Loader/DatabaseTranslationLoader.php

```diff
@@ -35,29 +35,40 @@ class DatabaseTranslationLoader implements LoaderInterface
 {
     /** @var EntityManagerInterface */
     protected $em;
```

Comment:
Not yours, but can't we rename this variable to `$entityManager` for a better comprehension?